.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Add space after this line. EOL

More on this here => https://adaptivepatchwork.com/2012/03/01/mind-the-end-of-your-line/#:~:text=will%20be%20LF%20.-,core.,to%20denote%20end%20of%20line.

Reply:
Fixed.